### ISSUE TITLE
AuditLogListenerUtil.getPersistentValue() always returns null

### ISSUE THREAD

**Issue by jnunderwood** [robertoschwald/grails-audit-logging-plugin#190]
### Steps to Reproduce

1. Run an app using this plugin and make changes to an Auditable domain object

### Expected Behaviour

The Audit Log should contain the old value.

### Actual Behaviour

The Audit Log's old value is null.

### Environment Information

- **Operating System**: Linux
- **GORM Version:** 6.1.11
- **Grails Version (if using Grails):** 3.3.9
- **JDK Version:** 1.8.0_212

### Details

In the master branch (version 3.0.5-SNAPSHOT), it seems that the `AuditLogListenerUtil.getPersistentValue()` method is the culprit.

**Comment by jnunderwood** [comment]
Workaround (fix?)... Change Line 75 in `AuditLogListenerUtil.getPersistentValue()`:

```groovy
        property instanceof ToMany ? "N/A" : ((GormEntity)domain).getPersistentValue(propertyName)
```

to

```groovy
        property instanceof ToMany ? "N/A" : ((GormEntity)domain).getOriginalValue(propertyName)
```

After making this change, the built-in tests complete without failure (and my own app works as expected).

**Comment by robertoschwald** [comment]
fixed in 3.0.5
